source/Application/Model/Article.php

```diff
@@ -488,6 +488,10 @@ public function __construct($aParams = null)
                 $this->$sParam = $mValue;
             }
         }
+        
+        // move this and the following dependencies to the constructor and do not use getters inside of methods
+        $this->config = $this->getConfig();
```

Comment:
also the following are perfect candidates:

\OxidEsales\Eshop\Core\Registry::getLang()
\OxidEsales\Eshop\Core\Registry::getUtils()
\OxidEsales\Eshop\Core\DatabaseProvider::getDb()